starknet/utils/data_transformer/data_transformer.py

```diff
@@ -99,6 +102,20 @@ def from_python(self, cairo_type, name, value):
 
     def to_python(self, cairo_type, name, values) -> (dict, CairoData):
         definition = self._definition(cairo_type)
+        (struct_name, *_) = definition.full_name.path
+
+        is_uint_256 = (
+            struct_name == "Uint256"
+            and len(definition.members.items()) == 2
+            and definition.members.get("low")
+            and definition.members.get("high")
+            and isinstance(definition.members["low"].cairo_type, TypeFelt)
+            and isinstance(definition.members["high"].cairo_type, TypeFelt)
+        )
+
+        if is_uint_256:
+            low, high = values
+            return UInt256((high << 128) + low), values
```

Comment:
You need to return values without 2 elements that you've read. If you had more values `low, high = values` would raise.

Reply:
Please add uint256 test to mixed params test, this way we'll be sure it works well with other types.